server/src/packages/core/index.ts

```diff
@@ -1,6 +1,38 @@
-import * as t from 'io-ts';
+import {string, refinement, TypeOf } from 'io-ts';
 
 export const uuidCheck = /^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$/i;
-export const UUIDRefinement = t.refinement(t.string, (str: string) => uuidCheck.test(str), 'Uuid');
+export const UUIDRefinement = refinement(string, (str: string) => uuidCheck.test(str), 'Uuid');
 
-export type Uuid = t.TypeOf<typeof UUIDRefinement>;
+// tslint:disable:max-classes-per-file
```

Comment:
Why is this declared here? 
Should it not be in a `UUID.ts` class file?

Reply:
We could do - is that your preference?